src/interpreter/Interpreter.ts

```diff
@@ -281,12 +281,6 @@ export class Interpreter {
     return this.gpu
   }
 
-  public destroy() {
-    if (this.gpu) {
-      this.gpu.destroy()
```

Comment:
I'm not sure about removing this one. GPU.js docs say that we should run a proper cleanup: https://github.com/gpujs/gpu.js/#cleanup

Reply:
ok, this should be then delegated to the main-class `destroy`.